backend/src/telegram/middlewares/telegram.middleware.ts

```diff
@@ -352,21 +345,16 @@ export const InitTelegramMiddleware = (
    */
   const setCampaignCredential = async (
     req: Request,
-    res: Response,
-    next: NextFunction
-  ): Promise<Response | void> => {
-    try {
-      const { campaignId } = req.params
-      const { credentialName } = res.locals
-      if (!credentialName) {
-        throw new Error('Credential does not exist')
-      }
-
-      await TelegramService.setCampaignCredential(+campaignId, credentialName)
-      return res.json({ message: 'OK' })
-    } catch (err) {
-      next(err)
+    res: Response
+  ): Promise<Response> => {
+    const { campaignId } = req.params
+    const { credentialName } = res.locals
+    if (!credentialName) {
+      throw new Error('Credential does not exist')
     }
+
+    await TelegramService.setCampaignCredential(+campaignId, credentialName)
+    return res.json({ message: 'OK' })
```

Comment:
[to clarify] is this supposed to be the campaignId? I see that some other endpoints return the campaignId

Reply:
No, the rest of the equivalent endpoints for other types were kept this way as well